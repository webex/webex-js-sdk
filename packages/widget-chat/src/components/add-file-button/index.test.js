import React from 'react';
import renderer from 'react-test-renderer';

import AddFileButton from '.';

describe(`AddFileButton component`, () => {
  let component, props;

  beforeEach(() => {
    props = {
      onClick: jest.fn(),
      onChange: jest.fn()
    };

  });

  it(`renders properly`, () => {
    component = renderer.create(
      <AddFileButton {...props} />
    );
    expect(component).toMatchSnapshot();
  });

});
