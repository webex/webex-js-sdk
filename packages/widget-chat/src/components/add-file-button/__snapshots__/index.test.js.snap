exports[`AddFileButton component renders properly 1`] = `
<button
  className="add-file-button"
  onClick={[Function mockConstructor]}>
  <span
    className="add-file-icon" />
</button>
`;
